File: QALY.java
import java.util.Scanner;

public class QALY{ 
  public static void main(String args[]){  
    Scanner kb= new Scanner(System.in);
    int n=kb.nextInt();
    if(1<=n && n<=100){
      double q,y;
      double sum=0;boolean ch=true;
      for(int i=0;i<n;i++){
        q=kb.nextDouble();
        y=kb.nextDouble();
        if(0<q && q<=1 && 0<y && y<=100)
          sum+=q*y;
        else{
          ch=false;
          break;
        }
      }
      if(ch) System.out.format("%.3f",sum);
    }
  }  
} 

File: qaly.py
n=int(input())
if(1<=n<=100):
  sum=0.0
  ch=True
  for i in range(0,n):
    l=input().split()
    q=float(l[0])
    y=float(l[1])
    if(0<q<=1 and 0<y<=100):
      sum+=q*y
    else:
      ch=False
      break
  if(ch): print('{0:.3f}'.format(sum))